supervisor/shared/safe_mode.c

```diff
@@ -77,6 +77,7 @@ safe_mode_t wait_for_safe_mode_reset(void) {
     common_hal_digitalio_digitalinout_construct(&boot_button, CIRCUITPY_BOOT_BUTTON);
     common_hal_digitalio_digitalinout_switch_to_input(&boot_button, PULL_UP);
     #endif
+    #ifndef CIRCUITPY_SKIP_SAFE_MODE_WAIT
```

Comment:
I'd rather have an early return and let the compiler remove the rest. It should be before line 69.

Reply:
Oops, you are completely right, there is even a comment there. I should have looked closer. Fixed now.